### PR TITLE
nodeup: create kubeconfig under admin or root

### PR DESCRIPTION
**Pull request by justinsb**
While the admin account is created on stock debian images, it isn't on
all of them.  Check admin first, then check root, and don't treat it as
an error if neither is found - this is only a convenience.

### CODE REVIEW

File: nodeup/pkg/model/kubectl.go [hunk 1]
```diff
@@ -74,29 +74,64 @@ func (b *KubectlBuilder) Build(c *fi.ModelBuilderContext) error {
...
-		switch b.Distribution {
-		case distros.DistributionJessie, distros.DistributionDebian9:
+		adminUser, adminGroup, err := b.findKubeconfigUser()
+		if err != nil {
+			return err
+		}
+
+		if adminUser != "" {
 			c.AddTask(&nodetasks.File{
 				Path:  "/home/admin/.kube/",
```
Comment (KashifSaadat) [inline nodeup/pkg/model/kubectl.go]: Should the path differ here based on the value of `adminUser`?

File: nodeup/pkg/model/kubectl.go [hunk 2]
```diff
@@ -74,29 +74,64 @@ func (b *KubectlBuilder) Build(c *fi.ModelBuilderContext) error {
...
 				Type:  nodetasks.FileType_Directory,
 				Mode:  s("0700"),
-				Owner: s("admin"),
-				Group: s("admin"),
+				Owner: s(adminUser),
+				Group: s(adminGroup),
 			})
 
 			c.AddTask(&nodetasks.File{
 				Path:     "/home/admin/.kube/config",
```
Comment (KashifSaadat) [inline nodeup/pkg/model/kubectl.go]: Same as above

**Comment by justinsb** [comment]
You're absolutely right @KashifSaadat  - thank you!

**Comment by KashifSaadat** [comment]
Thanks! :)

/lgtm

**Pull request merged**